web-platform/website/src/components/pages/account-settings/AccountSettingsPageContent.tsx

```diff
@@ -1,16 +1,16 @@
 import { Col, Container, Row } from "reactstrap";
 import { style } from "typestyle";
 
-import { Spacing } from "../../../styles";
-import { ContainerStyles } from "../../../styles/ContainerStyles";
+import { ScreenSize, Spacing, createResponsiveStyle } from "../../../styles";
```

Comment:
I suspect if you run npm run lint or npm run build it should lint the PR. I installed prettier in VSCode and using config from tools/typescript/.prettierrc to format as I save.